src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/books_storage";
+    private static final String DRIVER_URL = "com.mysql.cj.jdbc.Driver";
+    private static final Properties DB_PROPERTIES;
+    private static final String USER = "root";
+    private static final String PASSWORD = "55555Qwz!";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", USER);
+        DB_PROPERTIES.put("password", PASSWORD);
+
+        try {
+            Class.forName(DRIVER_URL);
+        } catch (ClassNotFoundException e) {
+            throw new DataProcessingException("Can't load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(DB_URL, DB_PROPERTIES);
+    }
```

Comment:
and here I would rather recommend doing a try-catch, to throw the exception closer to the reason since this is a bit more important